### ISSUE TITLE
Color for spock keywords unreadable in darcula

### ISSUE THREAD

**Issue by jschaf** [cholick/idea-spock-enhancements#30]
I had to change the color when using the darcula color scheme.  It would be nice if the color switched automatically.

**Comment by lobodpav** [comment]
Moreover, the colors sometimes change, sometimes not. See the attached screenshots.

**Correct color for dark scheme:**
![correct color](https://cloud.githubusercontent.com/assets/1579300/26341193/c7245f86-3f92-11e7-94e9-c6fa0cfeee76.png)

**Incorrect color for dark scheme:**
![incorrect color](https://cloud.githubusercontent.com/assets/1579300/26341194/c729156c-3f92-11e7-9a03-03879b9b7b41.png)

**Comment by norecha** [comment]
I just installed this plugin and had same issue with darcula. That blue is very blurry and irritating I had to disable the plugin, is there a workaround to change that color? I couldn't see any plugin entry in color menu

**Comment by lobodpav** [comment]
The workaround is quite simple. Once the plugin is installed, go to `Preferences`, click the `Spock` item in the root, and change the color to `#A9B7C6`.
<img width="489" alt="spock keyword color" src="https://user-images.githubusercontent.com/1579300/35169130-a3fb6d4c-fd5b-11e7-85c8-6f33e6fab9dc.png">

**Comment by norecha** [comment]
So that's where that option was, I was going through the code and created spockEnhancements.xml which has those 3 settings, and tried to put it in correct place, but gave up at some point. 
Turns out it is ~/Library/Preferences/IntelliJIdea14/options/spockEnhancements.xml

*My menu was little different, it was under "Other Settings>Spock".

Thanks

**Comment by chLemon** [comment]
I change it to FF8800( i love it), but it doesn't work

**Comment by lobodpav** [comment]
It seems like there is no active maintainer of this plugin 😞 Does anybody have a clue how to fix the plugin itself so that no such workarounds are necessary?

**Comment by lobodpav** [comment]
Not sure why the issue was closed when there is no commit fixing it.

**Comment by cholick** [comment]
I was cleaning up some GitHub stuff and realized I hadn't added "This isn't maintained" to the README. Sorry, thought I added that years ago. I probably should add that to the issue creation template too now that I think about it.

@lobodpav I bulk closed all the old stuff when I added that, but I realize I was looking at the creation date rather than last activity.

This plugin is going to break, though, before too long. It's using ~10 deprecated APIs and one that's actively schedule for removal.
